compel/arch/aarch64/src/lib/infect.c

```diff
@@ -3,6 +3,7 @@
 #include <sys/uio.h>
 #include <linux/elf.h>
 #include <compel/plugins/std/syscall-codes.h>
+#include <compel/plugins/std/syscall.h>
```

Comment:
Could you include stdlib.h to define NULL? It is better, because we need to avoid extra dependencies. If we will use this patch, this source file will be recompiled each time when compel/plugins/std/syscall.h is changed.